src/components/Publish/Pricing/Price.tsx

```diff
@@ -20,7 +23,13 @@ export default function Price({
   return (
     <div className={styles.price}>
       {values.pricing.type === 'free' ? (
-        <h4 className={styles.free}>Free</h4>
+        <h4 className={styles.free}>
```

Comment:
are we able to add the `className` onto the `Field`? Cause we output quite a bit of markup in the DOM in the end and semantically it shouldn't be wrapped in an `h4`

Reply:
yeah, that's right. I made some changes: change the h4 to div since it's worthless as such, and also swith the Field component with an Input one